pkg/controller/installation/validation.go

```diff
@@ -336,6 +336,25 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 				return fmt.Errorf("spec.calicoNetwork.containerIPForwarding is supported only for Calico CNI")
 			}
 		}
+
+		if instance.Spec.CalicoNetwork.Sysctl != nil {
+			// CNI tuning plugin
+			pluginData := instance.Spec.CalicoNetwork.Sysctl
+
+			// sysctl settings
+			allowedKeys := map[string]bool{
+				"net.ipv4.tcp_keepalive_intvl":  true,
+				"net.ipv4.tcp_keepalive_probes": true,
+				"net.ipv4.tcp_keepalive_time":   true,
+			}
+
+			for _, setting := range pluginData {
+				if allowedKeys[setting.Key] {
```

Comment:
I think this validation is incorrect.
I think currently it would fail validation if one of the allowedKeys were specified.

Reply:
yeah sry, you are right, the whole stuff was inverted and missing a "!". Thanks for the heads up